packages/language-support/src/highlighting/syntaxColouring/syntaxColouring.ts

```diff
@@ -300,3 +300,7 @@ export function applySyntaxColouring(
 
   return result;
 }
+
+export function clearCache() {
+  delete parserWrapper.parsingResult;
```

Comment:
Shouldn't this call parserWrapper.clearCache instead?

Reply:
It got in by mistake, thanks for catching it!